api/app/Models/PoolCandidate.php

```diff
@@ -589,10 +589,10 @@ public function scopeNotDraft(Builder $query): Builder
     public function getPoolCandidateStatusAttribute($candidateStatus)
     {
         // pull info
```

Comment:
any reason to keep this and other commented out code?

Reply:
I actually have no clue :sweat_smile: It was causing issues with my linter so I commented it out since it wasn't being used but I wasn't sure if it was planned on being brought back or not :woman_shrugging:

Reply:
Added ticket to remove:

 - https://github.com/GCTC-NTGC/gc-digital-talent/issues/11032